### PR TITLE
add AIO_DATABASE_HOST

### PR DESCRIPTION
**Pull request by szaimen**
Allow to get local ip-address of database container which allows to talk to it even in host mode (the container that requires this needs to be started first then)

For https://github.com/thanek/nextcloud-dlna/issues/1

### CODE REVIEW

**Comment by szaimen** [comment]
@thanek does this look good do you? :)

File: php/src/Docker/DockerActionManager.php [hunk 1]
```diff
@@ -403,6 +403,9 @@ public function CreateContainer(Container $container) : void {
                     } else {
                         $replacements[1] = '';
                     }
+                // Allow to get local ip-address of database container which allows to talk to it even in host mode (the container that requires this needs to be started first then)
+                } elseif ($out[1] === 'AIO_DATABASE_HOST') {
+                    $replacements[1] = gethostbyname('nextcloud-aio-database')[0];
```
[the remark this answers is not in the document]
Comment (szaimen) [reply php/src/Docker/DockerActionManager.php:408]: I thought it returns an array? In that case we only want the first element...

[the remark this answers is not in the document]
Comment (thanek) [reply php/src/Docker/DockerActionManager.php:408]: Afaik https://www.php.net/gethostbyname it returns string. But the https://www.php.net/gethostbynamel return an array ;)

[the remark this answers is not in the document]
Comment (szaimen) [reply php/src/Docker/DockerActionManager.php:408]: Ah okay you are right. Will adjust soon

[the remark this answers is not in the document]
Comment (szaimen) [reply php/src/Docker/DockerActionManager.php]: done

**Pull request merged**